src/main/kotlin/no/nav/syfo/oppfolgingsplan/OppfolgingsplanService.kt

```diff
@@ -81,6 +81,7 @@ class OppfolgingsplanService(
             dialogmeldingKode = DialogmeldingKode.KODE1.value,
             dialogmeldingTekst = null, // brukes ikke for oppfølgingsplan
             dialogmeldingVedlegg = oppfolgingsplan.oppfolgingsplanPdf,
+            kilde = "esyfo",
```

Comment:
Burde det være appnavn i stedet? Plutselig skal esyfo sende dialogmeldinger fra andre apper også 🤔

Reply:
Ja, kanskje. Men disse kommer inn "fra siden" siden vi har et api som esyfo bruker (i stedet for å legge bestilling på topic).

Reply:
Riktig 🤔 Evt `esfyo - oppfolgingsplan fra arbeidsgiver` eller noe sånt?

Reply:
Retter til ESYFO - sånn at vi kan skille det fra dialogmeldingene som sendes av oss selv (som får verdi SYFO).